<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label_acc_val">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>101</width>
     <height>19</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;微软雅黑&quot;;color: #fff;</string>
   </property>
   <property name="text">
    <string>null</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_acc">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>41</width>
     <height>19</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;微软雅黑&quot;;color: #fff;</string>
   </property>
   <property name="text">
    <string>账号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_name">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>70</y>
     <width>61</width>
     <height>19</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;微软雅黑&quot;;color: #fff;</string>
   </property>
   <property name="text">
    <string>用户：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_name_val">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>70</y>
     <width>71</width>
     <height>19</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;微软雅黑&quot;;
color: #fff;</string>
   </property>
   <property name="text">
    <string>null</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_group">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;微软雅黑&quot;;color: #fff;</string>
   </property>
   <property name="text">
    <string>套餐：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_group_val">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;微软雅黑&quot;;color: #fff;</string>
   </property>
   <property name="text">
    <string>null</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_flux_val">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>null</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_flux_val_2">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>160</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>null</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_update">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>46</x>
     <y>225</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #fff;
font: 10pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>Loading...</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>154</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>418</width>
     <height>270</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/image/show.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 16pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>快通流量查询工具 V3.1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_remainder">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>110</y>
     <width>61</width>
     <height>19</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;微软雅黑&quot;;color: #fff;</string>
   </property>
   <property name="text">
    <string>剩余：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_remainder_val">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>110</y>
     <width>71</width>
     <height>19</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;微软雅黑&quot;;
color: #fff;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>label_acc_val</zorder>
  <zorder>label_acc</zorder>
  <zorder>label_name</zorder>
  <zorder>label_name_val</zorder>
  <zorder>label_group</zorder>
  <zorder>label_group_val</zorder>
  <zorder>label_update</zorder>
  <zorder>progressBar</zorder>
  <zorder>label_flux_val_2</zorder>
  <zorder>label_flux_val</zorder>
  <zorder>label_2</zorder>
  <zorder>label_remainder</zorder>
  <zorder>label_remainder_val</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
